### ISSUE TITLE
#3 What is learning? What is Deep learning? (학습이란? 깊은 학습이란?)

### ISSUE THREAD

**Issue by Jo-won** [Jo-won/self-study-questions#3]
###** Learning**

Learning is a method that makes artificial neural network.
Then, MLP learning is to find Θ(u,v) if training set , X = { (x_1,t_1), (x_2,t_2), ... (x_N,t_N)  }, is given.
In (x,t), x is a feature vector and t is a object vector, what we want to find. If x equals to weight vector, t is (-1,-1,-1,..1,,,-1) (only t_j==1)
When training set ,X, is given o shows to output. So, we can tell error function,E(Θ)=0.5*sum of (t_i - o_i)^2 i is 1->m. In other words, finding Θ is 'learning'.
Learning algorithm have to go in direction of decreasing error. So, we can use 'gradient descent' to revise Θ. We know the direction of gradient, but we don't know the scale of gradient. Thus, ρ ,learning rate, is multiplied in gradients. Through this method, 'error back-propagation algorithm' is made. 
This algorithm calculates errors to the sequential order. And then, weight is revised to direction of decreasing error in reverse order.
If learning ends, sample recognition ends in one sequentially calculation.
In learning, many time must be used for sequentially calculation and error back-propagation. But operation of recognition is very fast since only one sequentially calculation.
 
**### Deep learning**

If learning is just 'learning' and 'classification' result from features constructed by human, deep learning is concept that features are also constructed by oneself not only learning's that.

-CNN (Convolutional Neural Network)
CNN is started from idea that learning's role is to construct mask oneself. So, CNN has a feature extraction process. First of all CNN has 7 layers. As i said before, from the front to the sixth of all layers are the hidden layer and from the front to the fifth of all layers are the feature extraction process. There are consisted to C1-S2-C3-S4-C5-F6-O.
C is a convolution layer, S is a downsampling layer, F likes to hidden layer of MLP, O is a output layer and  number means sequence of hidden layers.
In C, image is passed in mask and negative values are mapped 0 and positive values are maintained. In other words, activated features is given to next process. At this time, many feature maps are formed and each feature maps have each masks. We call this 'weight sharing'
In S said pooling, the downsampling is done because of decreasing size of operating.

In addition to CNN, Deep Neural Networks expanded MLP and RBM (Restricted Boltzmann Machine) are considered.
As I said before, Neural networks had hidden layers more than two are called 'Deep Neural Networks' and what is learned is called 'Deep learning.'
